app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/multipledisposals/asset_type_for_non_uk_residents.scala.html

```diff
@@ -72,56 +75,78 @@
   }
 }
 
-@options = @{
-    List(
-      RadioOption(
-        label = messages(s"$key.Residential"),
-        content = None,
-        optionHelpText = Some(Html(messages(s"$key.Residential.helpText")))
-      ),
-      RadioOption(
-        label = messages(s"$key.NonResidential"),
-        content = None,
-        optionHelpText = Some(Html(messages(s"$key.NonResidential.helpText")))
-      ),
-      RadioOption(
-        label = messages(s"$key.MixedUse"),
-        content = None,
-        optionHelpText = Some(Html(messages(s"$key.MixedUse$userKey.helpText")))
-      ),
-      RadioOption(
-        label = messages(s"$key$userKey.IndirectDisposal"),
-        content = None,
-        optionHelpText = Some(Html(messages(s"$key.IndirectDisposal.helpText")))
-      )
-    )
-}
-
-@mainTemplate(title = title, userType = request.userType, hasErrors = hasErrors) {
-
-    @backLinkComponent(backLink)
+@selected = @{form.value.map(_.map(toIndex)).getOrElse(List.empty)}
 
-    @if(hasErrors) {
-        @errorSummary(form, Some(s"$key$userKey"))
-    }
+@checkBoxes = @{List(
+  CheckboxItem(
+    id = Some(messages(s"$key-0")),
+    content = Text(messages(s"$key.Residential")),
+    hint = Some(Hint(content = Text(messages(s"$key.Residential.helpText")))),
+    value = "0",
+    checked = selected.contains(0)
```

Comment:
instead of using `selected` we can evaluate the form value in each item, for example:
```suggestion
        checked = form.value match {
      case Some(a: List[AssetType]) => a.contains(AssetType.Residential)
      case _  => false
    }
```

Reply:
Done